framework/src/main/java/org/checkerframework/common/value/ValueQualifierHierarchy.java

```diff
@@ -478,4 +476,54 @@ public boolean isSubtype(AnnotationMirror subAnno, AnnotationMirror superAnno) {
         return false;
     }
   }
+
+  // TODO: Move (part?) of this into plume-util's RegexUtil.
+  /**
+   * Return the strings such that any one of the regexes matches it.
+   *
+   * @param strings a collection of strings
+   * @param regexes a collection of regular expressions
+   * @return the strings such that any one of the regexes matches it
+   */
+  private List<String> anyMatches(List<String> strings, List<@Regex String> regexes) {
+    List<Pattern> patterns = new ArrayList<>(regexes.size());
+    for (String regex : regexes) {
```

Comment:
I think this could be replaced with   `List<Pattern> patterns = CollectionsPlume.mapList(Pattern::compile, regexes);`